src/numerics/IdasIntegrator.cpp

```diff
@@ -350,11 +350,6 @@ void IdasIntegrator::applyOptions()
         #else
             m_linsol_matrix = SUNDenseMatrix(N, N);
         #endif
-        #if SUNDIALS_VERSION_MAJOR >= 6
-            m_linsol_matrix = SUNDenseMatrix(N, N, m_sundials_ctx.get());
-        #else
-            m_linsol_matrix = SUNDenseMatrix(N, N);
-        #endif
```

Comment:
Is this still necessary for sundials 5.0?

Reply:
Yes, the "context" object was only introduced in Sundials 6.0.

Reply:
Right, sorry, that wasn't clear. It looks like this code is removed without an equivalent replacement. Maybe I missed something though...

Reply:
Ah, sorry, I misread your comment. And this is a bit different from the other removals -- this is removing a redundant initialization of `m_linsol_matrix`, identical to the 5 preceding lines. I did this as a separate commit (d22bab7bccfe136eae3fc15c2fc71ca93bb20d6d) but that's not apparent when looking at the whole PR.

Reply:
Awesome thanks for the context! My initial comment was totally unclear, no wonder you misunderstood 😁